src/cim/components/Agent/AgentsSelectionHostCountAlerts.tsx

```diff
@@ -2,37 +2,44 @@ import React from 'react';
 import { Alert, AlertGroup, AlertVariant } from '@patternfly/react-core';
 import ShortCapacitySummary from '../ClusterDeployment/ShortCapacitySummary';
 import { AgentK8sResource } from '../../types';
+import { TFunction } from 'i18next';
 
-const getHostCountWarningText = (hostsSelected: number) => {
+const getHostCountWarningText = (hostsSelected: number, t: TFunction) => {
   switch (hostsSelected) {
     case 0:
-      return 'No host is selected.';
+      return t<string>('ui:No host is selected.');
     case 1:
-      return 'Only 1 host is selected.';
+      return t<string>('ui:Only 1 host is selected.');
   }
-  return `Only ${hostsSelected} hosts are selected.`;
+  return t<string>('ui:Only {{hostsSelected}} hosts are selected.', { hostsSelected });
 };
 
 type AgentsSelectionHostCountAlertsProps = {
   matchingAgentsCount: number;
   selectedAgents: AgentK8sResource[];
   targetHostCount: number;
+  t: TFunction;
 };
 
 const AgentsSelectionHostCountAlerts = ({
   matchingAgentsCount,
   selectedAgents,
   targetHostCount,
+  t,
 }: AgentsSelectionHostCountAlertsProps) => {
   const selectedAgentsCount = selectedAgents.length;
   return (
     <AlertGroup>
       {selectedAgentsCount === targetHostCount && (
         <Alert
           variant={AlertVariant.success}
-          title={`${selectedAgentsCount} ${
-            selectedAgentsCount === 1 ? 'host' : 'hosts'
-          } selected out of ${matchingAgentsCount} matching.`}
+          title={`{selectedAgentsCount}`
+            .concat(selectedAgentsCount === 1 ? t<string>('ui:host') : t<string>('ui:hosts'))
+            .concat(
+              t<string>('ui: selected out of matchingAgentsCount matching.', {
+                matchingAgentsCount,
+              }),
+            )}
```

Comment:
This is incorrect usage of i18n
We cannot construct strings (no concat etc)
Also for plurals use https://www.i18next.com/translation-function/plurals